ibm/mas_devops/roles/dro/templates/subscription/dro_subscription.yml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: ibm-data-reporter-operator
   namespace: redhat-marketplace
   labels:
-    managed-by: mas
+    instance: dro
```

Comment:
This should be `app.kubernetes.io/instance` per https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/

Reply:
done. updated

Reply:
Updated and instead of _app.kubernetes.io/instance_ added _app.kubernetes.io/name_ label